README.md

```diff
@@ -71,10 +71,10 @@ At this point, you should be ready to start your app in development mode.
 3. As long as you leave `SUPPRESS_SELF_INVITE=` blank in your `.env` you should be able to invite yourself from the homepage.
    - If you DO set that variable, then spoke will be invite-only and you'll need to generate an invite. Run:
      ```
-     echo "INSERT INTO invite (hash,is_valid) VALUES ('abc', 1);" |sqlite3 mydb.sqlite
+     echo "INSERT INTO invite (hash,is_valid) VALUES ('E4502B28-301E-4E63-9A97-ACA14E8160C8', 1);" |sqlite3 mydb.sqlite
      # Note: When doing this with PostgreSQL, you would replace the `1` with `true`
      ```
-   - Then use the generated key to visit an invite link, e.g.: http://localhost:3000/invite/abc. This should redirect you to the login screen. Use the "Sign Up" option to create your account.
+   - Then use the generated key to visit an invite link, e.g.: http://localhost:3000/invite/E4502B28-301E-4E63-9A97-ACA14E8160C8. This should redirect you to the login screen. Use the "Sign Up" option to create your account.
```

Comment:
Does this actually name the organization "abc" or does it end up naming the org `E4502B28-301E-4E63-9A97-ACA14E8160C8`?

I'm a little concerned to see the hash in the invite url. I'm pretty sure when I ran `echo "INSERT INTO invite (hash,is_valid) VALUES ('abc', 1);"` on postgres it *did* actually generate an org called "abc"

Reply:
it shouldn't create an org called 'abc' (or the hash) -- `hash` is a column in the `invite` table -- so it would be very odd if it populated the org name.

@ibrand Can you confirm that?

Reply:
I can't claim to be super experienced with Spoke's codebase but i found that the hash here was not associated with the company name, as once I signed up using this hash, I was then given the prompt for company name and proceeded to create the first login from there.

The hash I provided was randomly generate just from the `uuidgen` cli command and is not the hash of `abc`.

Reply:
I reran this and was mistaken! My bad here. I was misremembering the last time we used this command. This *does* only impact the invite table and, like @kylehotchkiss said, it works when it's encoded